build.gradle

```diff
@@ -1,24 +1,31 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.3.50'
+
     repositories {
         google()
         jcenter()
-        
+
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.5.1'
         classpath AndroidDependencies.kotlinPlugin
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }
 
+plugins {
+    id "com.diffplug.gradle.spotless" version "3.14.0"
```

Comment:
Let's try on this too, if not done already.

Reply:
I tried but it is giving error.